draft-yasskin-wpack-bundled-exchanges.md

```diff
@@ -134,7 +135,7 @@ metadata, and if one matches, load that request's response.
 This takes the bundle's stream and returns either an error (where an error is a
 "format error" or a "version error"), an error with a fallback URL (which is
 also the primaryUrl when the bundle parses successfully), or a map ({{INFRA}})
-of metadata containing at least keys named:
+of metadata containing keys named:
```

Comment:
I think you can just remove the `manifest` item below (L160-164) and leave the other text in this section unchanged. Section 3.3.2 has all the information about the manifest section.

Reply:
done